code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -33,9 +33,10 @@
 			if(M.confused < drunk_value)
 				M.confused += 3
 		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1)
-	..()
-	return
+			M.adjustToxLoss(1, 0)
+			. = 1
+	return ..() | .
```

Comment:
did you actually mean ```||```? I don't quite see the use for ```|``` here (you'd be doing a binary or of 1 and whatever ..() returns, as opposed to a logical one)